modules/tinymce/src/core/main/ts/api/util/Color.ts

```diff
@@ -11,7 +11,7 @@
  * @deprecated
  * @class tinymce.util.Color
  * @private
- * @summary Color has been deprecated and marked for removal in TinyMCE 6.0.
+ * @summary Color has been deprecated in TinyMCE 5.10 and has been marked for removal in TinyMCE 6.0.
```

Comment:
Is there a way to make this consistent with the others? Move the deprecation message up into the description, instead of inside an `@summary`?

Reply:
See the other comment.